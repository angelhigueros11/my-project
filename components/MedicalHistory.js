import React from "react";
import HistoryDetail from "./HistoryDetail";
import { KeyboardAvoidingView,StyleSheet, Text,TextInput, View,SafeAreaView,Image,TouchableOpacity,Platform,ScrollView } from 'react-native';

export default function MedicalHistory(){
        
    return (
        <View style={styles.container}>
            <View style={styles.scrollView}>
                <ScrollView>
                    <HistoryDetail title='Operación Cabeza' date='02/01/2019'/>
                    <HistoryDetail title='Golpe espalda' date='20/06/2019'/>
                    <HistoryDetail title='Operación Amigdalas' date='02/07/2020'/>
                    <HistoryDetail title='Operación pierna' date='02/01/2017'/>
           
                    <HistoryDetail title='Operación Cabeza' date='02/01/2019'/>
                    <HistoryDetail title='Golpe espalda' date='20/06/2019'/>
                    <HistoryDetail title='Operación Amigdalas' date='02/07/2020'/>
                    <HistoryDetail title='Operación pierna' date='02/01/2017'/>
           
                </ScrollView>
            </View>
        </View>
    )
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        width: '100%',
        alignItems: 'center',
        justifyContent: 'center',
        backgroundColor: '#f5f5f5',
    },scrollView: {
        width: '90%',
        height: '95%',
        backgroundColor: '#fff',
    
    }
});