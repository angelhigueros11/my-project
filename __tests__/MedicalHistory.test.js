import React from 'react';
import renderer from 'react-test-renderer';
import MedicalHistory from '../components/MedicalHistory';

describe('Pruebas en <HistoryDetail />', () => {
    it('Renders correctly', () => {
        const tree = renderer.create(<MedicalHistory />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    
});