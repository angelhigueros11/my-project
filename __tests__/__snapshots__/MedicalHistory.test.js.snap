// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <MedicalHistory /> Renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#f5f5f5",
      "flex": 1,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "height": "95%",
        "width": "90%",
      }
    }
  >
    <RCTScrollView>
      <View>
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "backgroundColor": "#fff",
              "borderBottomColor": "#d7d6d6",
              "borderBottomWidth": 2,
              "height": 100,
              "opacity": 1,
              "padding": 10,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Titulo:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              Operación Cabeza
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Fecha:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              02/01/2019
            </Text>
          </View>
          <View
            style={
              Array [
                Object {
                  "flexDirection": "row",
                  "height": "auto",
                  "marginTop": 5,
                  "overflow": "hidden",
                },
                Object {
                  "marginTop": "auto",
                  "position": "relative",
                },
              ]
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Descripción:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              lorem ipsum dolor sit amet,...
            </Text>
          </View>
        </View>
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "backgroundColor": "#fff",
              "borderBottomColor": "#d7d6d6",
              "borderBottomWidth": 2,
              "height": 100,
              "opacity": 1,
              "padding": 10,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Titulo:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              Golpe espalda
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Fecha:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              20/06/2019
            </Text>
          </View>
          <View
            style={
              Array [
                Object {
                  "flexDirection": "row",
                  "height": "auto",
                  "marginTop": 5,
                  "overflow": "hidden",
                },
                Object {
                  "marginTop": "auto",
                  "position": "relative",
                },
              ]
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Descripción:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              lorem ipsum dolor sit amet,...
            </Text>
          </View>
        </View>
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "backgroundColor": "#fff",
              "borderBottomColor": "#d7d6d6",
              "borderBottomWidth": 2,
              "height": 100,
              "opacity": 1,
              "padding": 10,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Titulo:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              Operación Amigdalas
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Fecha:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              02/07/2020
            </Text>
          </View>
          <View
            style={
              Array [
                Object {
                  "flexDirection": "row",
                  "height": "auto",
                  "marginTop": 5,
                  "overflow": "hidden",
                },
                Object {
                  "marginTop": "auto",
                  "position": "relative",
                },
              ]
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Descripción:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              lorem ipsum dolor sit amet,...
            </Text>
          </View>
        </View>
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "backgroundColor": "#fff",
              "borderBottomColor": "#d7d6d6",
              "borderBottomWidth": 2,
              "height": 100,
              "opacity": 1,
              "padding": 10,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Titulo:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              Operación pierna
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Fecha:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              02/01/2017
            </Text>
          </View>
          <View
            style={
              Array [
                Object {
                  "flexDirection": "row",
                  "height": "auto",
                  "marginTop": 5,
                  "overflow": "hidden",
                },
                Object {
                  "marginTop": "auto",
                  "position": "relative",
                },
              ]
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Descripción:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              lorem ipsum dolor sit amet,...
            </Text>
          </View>
        </View>
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "backgroundColor": "#fff",
              "borderBottomColor": "#d7d6d6",
              "borderBottomWidth": 2,
              "height": 100,
              "opacity": 1,
              "padding": 10,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Titulo:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              Operación Cabeza
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Fecha:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              02/01/2019
            </Text>
          </View>
          <View
            style={
              Array [
                Object {
                  "flexDirection": "row",
                  "height": "auto",
                  "marginTop": 5,
                  "overflow": "hidden",
                },
                Object {
                  "marginTop": "auto",
                  "position": "relative",
                },
              ]
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Descripción:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              lorem ipsum dolor sit amet,...
            </Text>
          </View>
        </View>
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "backgroundColor": "#fff",
              "borderBottomColor": "#d7d6d6",
              "borderBottomWidth": 2,
              "height": 100,
              "opacity": 1,
              "padding": 10,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Titulo:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              Golpe espalda
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Fecha:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              20/06/2019
            </Text>
          </View>
          <View
            style={
              Array [
                Object {
                  "flexDirection": "row",
                  "height": "auto",
                  "marginTop": 5,
                  "overflow": "hidden",
                },
                Object {
                  "marginTop": "auto",
                  "position": "relative",
                },
              ]
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Descripción:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              lorem ipsum dolor sit amet,...
            </Text>
          </View>
        </View>
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "backgroundColor": "#fff",
              "borderBottomColor": "#d7d6d6",
              "borderBottomWidth": 2,
              "height": 100,
              "opacity": 1,
              "padding": 10,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Titulo:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              Operación Amigdalas
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Fecha:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              02/07/2020
            </Text>
          </View>
          <View
            style={
              Array [
                Object {
                  "flexDirection": "row",
                  "height": "auto",
                  "marginTop": 5,
                  "overflow": "hidden",
                },
                Object {
                  "marginTop": "auto",
                  "position": "relative",
                },
              ]
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Descripción:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              lorem ipsum dolor sit amet,...
            </Text>
          </View>
        </View>
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "backgroundColor": "#fff",
              "borderBottomColor": "#d7d6d6",
              "borderBottomWidth": 2,
              "height": 100,
              "opacity": 1,
              "padding": 10,
              "width": "100%",
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Titulo:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              Operación pierna
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "height": "auto",
                "marginTop": 5,
                "overflow": "hidden",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Fecha:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              02/01/2017
            </Text>
          </View>
          <View
            style={
              Array [
                Object {
                  "flexDirection": "row",
                  "height": "auto",
                  "marginTop": 5,
                  "overflow": "hidden",
                },
                Object {
                  "marginTop": "auto",
                  "position": "relative",
                },
              ]
            }
          >
            <Text
              style={
                Object {
                  "color": "#616161",
                }
              }
            >
              Descripción:
            </Text>
            <Text
              style={
                Object {
                  "marginLeft": 10,
                }
              }
            >
              lorem ipsum dolor sit amet,...
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
